securesystemslib/exceptions.py

```diff
@@ -113,3 +113,8 @@ class StorageError(Error):
   """Indicate an error occured during interaction with an abstracted storage
      backend."""
```

Comment:
While you are on this file, can you fix this typo as well:
```suggestion
  """Indicate an error occurred during interaction with an abstracted storage
     backend."""
```